packages/orchestrator/lib/clients/processor.integration.test.ts

```diff
@@ -37,52 +38,61 @@ describe('OrchestratorProcessor', () => {
 
     afterAll(async () => {
         scheduler.stop();
+        await setTimeout(100); // wait for the scheduler to stop
```

Comment:
unrelated. but sometimes db will be cleared before the scheduler is actually stopped